CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file. This projec
 
 ## MASTER
 
+## 2.6.1 - 2021-08-02
```

Comment:
All the changes on this file should be reverted because there will be no release and these changes are not relevant to the users.

Reply:
Done